pkgs/development/tools/profiling/systemtap/default.nix

```diff
@@ -0,0 +1,56 @@
+{ fetchurl, linuxPackages, makeWrapper, runCommand, fetchgit
+, elfutils, pkgconfig, gettext, stow, gnumake }:
+let
+  ## fetchgit info
+  url = git://sourceware.org/git/systemtap.git;
```

Comment:
I see, there is https://gitlab.com/fche/systemtap.git
it's not as up-to-date as the `git:` uri, that might not matter much for packaging, though

do we prefer `https` because of firewalls?